config/pluginspec.yaml

```diff
@@ -1613,12 +1618,15 @@ configuration:
     type: "entry"
     required: false
     documentation: "Physical path to jboss-cli."
+    label: Physical location of the jboss client script
   - name: "test_connection"
     type: "checkbox"
     required: false
     documentation: "\n                    Check if connection should be established\
       \ during config creation to validate credentials.\n                "
+    label: Check Connection?
```

Comment:
In new plugin configuration there should be no test connection checkbox.
https://cloudbees.atlassian.net/browse/BEE-18400

Reply:
Please take a look at EC-Git

Reply:
Oh, I see! I plan to fix it in BEE-18400, in this PR are only labels fixes.